ocrd/ocrd/cli/ocrd_tool.py

```diff
@@ -96,15 +96,23 @@ def ocrd_tool_tool_description(ctx):
 @ocrd_tool_tool.command('list-resources', help="List tool's file resources")
 @pass_ocrd_tool
 def ocrd_tool_tool_list_resources(ctx):
-    Processor(None, ocrd_tool=ctx.json['tools'][ctx.tool_name],
-              list_resources=True)
+    class BashProcessor(Processor):
+        @property
+        def moduledir(self):
+            return os.path.dirname(ctx.filename)
```

Comment:
But will this work with the `SHAREDIR=...` convention we use for `make install` of bashlib processors? In that case, the executable might be installed to `$HOME/.local/bin`, but the resources to `$HOME/.local/share/<projectname>`?

Reply:
Yes, it does work – as long as the resources are in the same directory as the ocrd-tool.json file (which is the case for all existing bashlib processors we have). It does not matter where the executable itself is, or which directory the resources go to (venv or XDG or whatever `make install` does).